packages/swap/src/utils/getPoolQuote.ts

```diff
@@ -114,7 +118,7 @@ export default async function getPoolQuote({
       destAsset,
       boosted: Boolean(boostFeeBps),
     }),
-    estimatedPrice: getPrice(swapInputAmount, srcAsset, egressAmount, destAsset),
+    estimatedPrice: getPrice(swapInputAmount, srcAsset, swapOutputAmount, destAsset),
```

Comment:
the `estimatedPrice` should be the price at which the assets are swapped at the amm level to allow for calculating a fok min price like this:`quote.estimatedPrice * 0.99`